### PR TITLE
New package: goatcounter-1.2.0

### LINKED ISSUE #22087: Package Request: GoatCounter
https://github.com/zgoat/goatcounter

> Simple web analytics. No tracking of personal data.

License: [EUPL](https://github.com/zgoat/goatcounter/blob/master/LICENSE)

### PR DESCRIPTION
**Pull request by anjandev**
closes  #22087

### CODE REVIEW

**Comment by Anachron** [comment]
Maybe a service file would be useful? Also running as an unprivilidged user may be good.

File: srcpkgs/goatcounter/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+go_import_path=zgo.at/goatcounter
+short_desc="Easy web analytics. No tracking of personal data"
+maintainer="Anjandev Momi <anjan@momi.ca>"
+license="EUPL-1.2"
+homepage="https://github.com/zgoat/goatcounter"
+distfiles="https://github.com/zgoat/goatcounter/archive/v${version}.tar.gz"
+checksum=505f8c5848bb368e18c0a041fcbeb87f540813a7455d9a67cdce5a5f25d799e7
+
+do_build() {
+	go build ./cmd/goatcounter
```
Comment (Johnnynator) [inline srcpkgs/goatcounter/template:16]: Should be possible to be handled by setting `go_package="${go_import_path}/cmd/goatcounter"` Also should allow to remove `do_install`

File: srcpkgs/goatcounter/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'goatcounter'
+pkgname=goatcounter
+version=1.2.0
+revision=1
+wrksrc=${pkgname}-${version}
```
Comment (sgn) [inline srcpkgs/goatcounter/template:5]: Isn't this set by default?

File: srcpkgs/goatcounter/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+build_style=go
+go_import_path=zgo.at/goatcounter
+go_package="${go_import_path}/cmd/goatcounter"
+short_desc="Easy web analytics. No tracking of personal data"
```
Comment (sgn) [inline srcpkgs/goatcounter/template:9]: I would change `short_desc` by
```
s/[.] No/ without/
```
I'm fine with current `short_desc`, though.

**Comment by Anachron** [comment]
Hows it going here? Is something missing?

**Pull request closed without merging**